svcomp/bench/toInstallDir/data/index.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
```

Comment:
Please add comments explaining what this file does. Also, is python3 now required?